### ISSUE TITLE
@babel/helper@v7.3.1 objectSpread implemention makes error

### ISSUE THREAD

**Issue by 4x4l** [babel/babel#9382]
## Bug Report

**Current Behavior**
```javascript
{
 {},
 undefined,
 undefined
}
```
now it will throw error
`TypeError: undefined is not an object (evaluating 'Object.getOwnPropertyDescriptors(arguments[i])')`

```js
var your => (code) => here;
```

**Expected behavior/code**
```javascript
{
 {},
 undefined,
 undefined
}
// return {}, no error
```

**Babel Configuration (.babelrc, package.json, cli command)**

```js
{
  "@babel/runtime": "^7.2.0",
}
```

**Environment**
- Babel version(s): [7.2.2]
- Node/npm version: [Node 8/npm 5]
- OS: [OSX 10.14.1 ]
- Monorepo: [no]
- How you are using Babel: [`loader`]

**Possible Solution**
correct implemention

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by loganfsmyth** [comment]
There was a regression causing this with runtime 7.3, but it should be resolved with 7.3.1

**Comment by nicolo-ribaudo** [comment]
This regression has been fixed in v7.3.1
